CONTRIBUTING.md

```diff
@@ -28,6 +28,66 @@ use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
 
+## Pull Request Title Convention
+
+We enforce a PR title convention to quickly indicate type and scope of a PR.
+We also parse it programmatically for changelog generation.
+
+PR titles should
+* be a user facing description of the change
+* be [conventional](https://www.conventionalcommits.org/en/v1.0.0/)
+* append fixed issue(s) at the end (if any)
+
+Examples:
+* `"fix(ui): fixes empty page. Fixes #1234"`
+* `"feat(backend): configurable service account. Fixes #1234, fixes #1235"`
+* `"chore: refactor some files"`
+* `"test: fix CI failure. Part of #1234"` // use "part of" when a PR is an work item of some issue, but shouldn't close the issue when merged.
+
+A quick summary of the conventions we care about:
+### PR Title Structure
+```
```

Comment:
Add a sentence like the following:

PR titles should use the following structure.